phoonnx_train/optispeech/text/__init__.py

```
@@ -0,0 +1,46 @@
+        self.tokenizer = tokenizer_cls(add_blank=add_blank, add_bos_eos=add_bos_eos, normalize_text=normalize_text)
+        self.num_languages = len(languages)
+        self.is_multi_language = self.num_languages > 1
+        self.default_language = languages[0].strip().lower()
+
+    def __call__(self, text, lang, split_sentences: bool = False):
+        # handle special value
+        if lang is None:
+            lang = self.default_language
+        lang = lang.strip().lower()
+        if lang not in self.languages:
+            raise ValueError(f"Language {lang} does not exist in the supported language list.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)